backends/tfhe-cuda-backend/cuda/src/pbs/programmable_bootstrap_classic.cu

```diff
@@ -650,9 +649,8 @@ void cuda_programmable_bootstrap_lwe_ciphertext_vector_64(
     int8_t *mem_ptr, uint32_t lwe_dimension, uint32_t glwe_dimension,
     uint32_t polynomial_size, uint32_t base_log, uint32_t level_count,
     uint32_t num_samples, uint32_t lut_count, uint32_t lut_stride) {
-  if (base_log > 64)
-    PANIC("Cuda error (classical PBS): base log should be > number of bits "
-          "in the ciphertext representation (64)");
```

Comment:
for both pbs<u32> and pbs<u64> we support base log up to 32.  in decomposition we have:

`      result[tid].x = (int32_t)res_re;
      result[tid].y = (int32_t)res_im;
`

Reply:
sorry misunderstood the change changes